mumd/src/audio.rs

```diff
@@ -60,19 +61,25 @@ pub enum NotificationEvents {
     Undeafen,
 }
 
+#[derive(Eq, PartialEq, Clone, Copy, Hash)]
+pub enum VoiceStream {
```

Comment:
Bad name. Should probably be something along the lines of `VoiceStreamType` since the name `VoiceStream` implies that it yields data, which it does not.